internal/services/paloalto/next_generation_firewall_vhub_local_rulestack_resource_test.go

```diff
@@ -151,9 +174,25 @@ resource "azurerm_palo_alto_next_generation_firewall_virtual_hub_local_rulestack
 }
 
 func (r NextGenerationFirewallVWanResource) requiresImport(data acceptance.TestData) string {
-	return fmt.Sprintf(`
+	if !features.FivePointOhBeta() {
+		return fmt.Sprintf(`
+%[1]s
 
+resource "azurerm_palo_alto_next_generation_firewall_virtual_hub_local_rulestack" "import" {
+  name                = azurerm_palo_alto_next_generation_firewall_virtual_hub_local_rulestack.test.name
+  resource_group_name = azurerm_palo_alto_next_generation_firewall_virtual_hub_local_rulestack.test.resource_group_name
+  rulestack_id        = azurerm_palo_alto_next_generation_firewall_virtual_hub_local_rulestack.test.rulestack_id
+  plan_id             = azurerm_palo_alto_next_generation_firewall_virtual_hub_local_rulestack.test.plan_id
 
+  network_profile {
+    virtual_hub_id               = azurerm_palo_alto_next_generation_firewall_virtual_hub_local_rulestack.test.network_profile.0.virtual_hub_id
+    network_virtual_appliance_id = azurerm_palo_alto_next_generation_firewall_virtual_hub_local_rulestack.test.network_profile.0.network_virtual_appliance_id
+    public_ip_address_ids        = azurerm_palo_alto_next_generation_firewall_virtual_hub_local_rulestack.test.network_profile.0.public_ip_address_ids
+  }
+}
+`, r.basic(data))
+	}
+	return fmt.Sprintf(`
```

Comment:
I'm not sure this is necessary since `plan_id` is an optional property, the existing config for the requires import test should still work?

Reply:
Yes. Test case works fine.